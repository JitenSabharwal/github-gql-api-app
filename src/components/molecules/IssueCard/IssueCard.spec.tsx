import Card, { Issue } from "./IssueCard"
import React from "react"
import { shallow } from "enzyme"

const cardMockData: Issue = {
  id: 1,
  createdAt: new Date(),
  title: "Sample Issue",
  description: "This is just for testing purpose",
  createdBy: "Jiten",
  repository: "Sample Test",
  commentCount: 15,
}
describe("Card", () => {
  let component: React.ReactElement
  beforeAll(
    () => (component = <Card issue={cardMockData} onShowMore={() => {}} />)
  )
  it("Should render a <div />", () => {
    const container = shallow(component)
  })
  it("Should render the layout", () => {
    const container = shallow(component)
    expect(container.getElements()).toMatchSnapshot()
  })
})
