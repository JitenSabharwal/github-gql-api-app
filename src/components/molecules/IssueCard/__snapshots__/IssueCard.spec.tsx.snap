// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Should render the layout 1`] = `
Array [
  <styled.div />,
]
`;
